storage/src/vespa/storage/bucketdb/bucketmanagermetrics.cpp

```diff
@@ -23,11 +25,28 @@ DataStoredMetrics::DataStoredMetrics(const std::string& name, metrics::MetricSet
     ready.logOnlyIfSet();
 }
 
-DataStoredMetrics::~DataStoredMetrics() { }
+DataStoredMetrics::~DataStoredMetrics() = default;
 
-BucketManagerMetrics::BucketManagerMetrics()
+BucketSpaceMetrics::BucketSpaceMetrics(const vespalib::string& space_name, metrics::MetricSet* owner)
+        : metrics::MetricSet("bucket_space", {{"name", space_name}}, "", owner),
```

Comment:
I think we should give dimension a more specific name. E.g 'bucketSpace'. Yes, we use camel case for dimension names.

Reply:
Renamed `name` to `bucketSpace`